publish.sh

```diff
@@ -25,7 +26,11 @@ done
 IMAGES="${CLUSTERS}"
 docker push ${TARGET_REPO}/gocli:${KUBEVIRTCI_TAG}
```

Comment:
@dhiller is a skopeo copy required here too?

Reply:
Good point :blush:

Reply:
@brianmcarey fixed, PTAL, thanks!